application/src/main/java/org/togetherjava/tjbot/features/reminder/RemindRoutine.java

```diff
@@ -154,15 +154,13 @@ Failed to send a reminder with (authorID '{}') skipping it. This can be due to a
 
         int failureAttempts = pendingReminder.getFailureAttempts() + 1;
         Instant remindAt = Instant.now().plus(1, ChronoUnit.MINUTES);
-        database.write(context -> context.newRecord(PENDING_REMINDERS)
-            .setId(pendingReminder.getId())
-            .setCreatedAt(Instant.from(pendingReminder.getCreatedAt()))
-            .setGuildId(pendingReminder.getGuildId())
-            .setChannelId(pendingReminder.getChannelId())
-            .setAuthorId(pendingReminder.getAuthorId())
-            .setRemindAt(remindAt)
-            .setContent(pendingReminder.getContent())
-            .setFailureAttempts(failureAttempts)
-            .insert());
+        database.write(context -> {
+            pendingReminder.setRemindAt(remindAt);
+            pendingReminder.setFailureAttempts(failureAttempts);
+            pendingReminder.insert();
+        });
+
+
+
```

Comment:
3 newlines seem a bit too much :P

Reply:
that's not spotless?

Reply:
no

Reply:
i don't understand, it's a pre-commit hook and it did what it was supposed to do `format the code` and i thought maybe its Spotless's doing.

Reply:
spotless isn't supposed to format ur code 100%, u can always use some edge case exploit, so thats y the review process is there, just remove the empty lines

Reply:
spotless didnt add those newlines, that must have been u. spotless isnt perfect, it cant remove the newlines unfortunately.